packages/jsts/src/rules/S109/cb.fixture.js

```diff
@@ -0,0 +1,33 @@
+function doSomething() {
+  for (let i = 0; i < 42; i++) { // Noncompliant, 4 is a magic number
```

Comment:
You meant something like that?
```suggestion
  for (let i = 0; i < 42; i++) { // Noncompliant {{No magic number: 42}}
```